src/Security/MemberAuthenticator/CookieAuthenticationHandler.php

```diff
@@ -39,70 +42,77 @@ class CookieAuthenticationHandler implements AuthenticationHandler
 
     /**
      * Get the name of the cookie used to track this device
-     *
-     * @return string
      */
-    public function getDeviceCookieName()
+    public function getDeviceCookieName(): string
     {
-        return $this->deviceCookieName;
+        return $this->deviceCookieName ?? '';
```

Comment:
```suggestion
        return $this->deviceCookieName;
```
Won't PHP complain about trying to access an un-initilised value?

Seems like it would be better to just give the property a default value of blank string

Reply:
No, it only does that for properties which are explicitly typed. But I guess I can make it explicitly string.

Reply:
Done